Functions.Templates/Templates/CosmosDBTrigger-CSharp-5.x/.template.config/template.json

```diff
@@ -0,0 +1,78 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Azure Function", "Trigger", "Cosmos DB"],
+  "name": "CosmosDBTrigger",
+  "identity": "Azure.Function.CSharp.CosmosDBTrigger.2.x",
```

Comment:
Is this supposed to be 2.x or should it match the extension version 4.x? Also, should the folder name match the extension version 4.x instead of 5.x?

Reply:
For the first question, I will defer that to @soninaren. 

For the folder name, ideally, yes, but I named it `-5.x` because I saw that there were already `-5.x` folders for PowerShell, JavaScript, and Python. Would you know why those were named `-5.x`? Ultimately, though, folder names don't really matter that much, what matters is they are properly referenced in the ItemTemplates.nupkg file

Reply:
I'm surprised to see we have folders named 5.x; the value is supposed to represent the host version I think - at least that's the case for the project templates. And for track two, there's a new folder for track 2 templates called "Templates-v2". 

I'll sync with @soninaren and see what we want to do about the state of the repo folders + identity

Reply:
I believe that for item templates, the suffix is supposed to represent the version of the extension, which is `5.x` for most track 2 Azure SDK extensions. CosmosDB is one of few exceptions where their track 2 extension is version `4.x`. Also, I want to clarify that here, when we say track 2, we're talking about the versions of the extensions themselves. My understanding is that `Templates-v2` refers to v2 of the template schema, regardless of extension or host version.

Reply:
That makes sense, thanks for the extra context